src/models/d2Model.ts

```diff
@@ -47,17 +49,17 @@ abstract class D2Model {
 }
 
 export class OrganisationUnitModel extends D2Model {
-    protected static metadataType = 'organisationUnit';
+    protected static metadataType = "organisationUnit";
```

Comment:
The current workflow has a lot of room for improvement. What do you think of adding a pre-push hook to prevent a push when there are uncommitted changes?

https://stackoverflow.com/questions/2831172/how-can-i-prevent-git-push-if-local-modifications-are-detected-including-untrac

Reply:
Maybe we can create a ticket to tackle both these types of configs as well as the TS type/module export configs?

Reply:
@matiasgarcia91 did you finally create a ticket for this?

Reply:
no, TS configs seem to be solved but I'll create one to unify configs between both apps, what do you think?

Reply:
Yep, go ahead!

Reply:
https://github.com/EyeSeeTea/metadata-synchronization/issues/38